### ISSUE TITLE
Need clearer distinction between WCAG notes and WCAG2ICT added notes

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#432]
#### In the success criteria (SCs)

It's not clear enough which notes are from WCAG and which notes are added by WCAG2ICT.

(1) When an SC does not have word substitution, and WCAG has one or more notes, and WCAG2ICT adds one or more notes, it can create the impression that the WCAG2ICT notes are meant to replace the WCAG notes. This is particularly a problem when both the WCAG and the WCAG2ICT notes are numbered, because the added WCAG2ICT notes repeat the same numbers as the WCAG notes.

Examples (partial list):

- [1.4.12 Text Spacing](https://w3c.github.io/wcag2ict/#text-spacing) — WCAG has notes 1 and 2; WCAG2ICT has notes 1, 2, 3, and 4
- [2.1.1 Keyboard](https://w3c.github.io/wcag2ict/#keyboard) — WCAG has notes 1 and 2; WCAG2ICT has notes 1, 2, and 3

(2) When an SC has both word substitution and added notes, it is not clear where the added notes begin. This is somewhat of an issue where WCAG has no notes. It is more of an issue when WCAG had one or more notes, and WCAG2ICT also adds one or more notes.

Examples (partial list):

- [1.4.2 Audio Control](https://w3c.github.io/wcag2ict/#audio-control) — WCAG has 1 note; WCAG2ICT adds 1 more
- [1.4.10 Reflow](https://w3c.github.io/wcag2ict/#reflow) — WCAG has 2 notes; WCAG2ICT adds 7 more
- [2.1.2 No Keyboard Trap](https://w3c.github.io/wcag2ict/#no-keyboard-trap) — WCAG has 1 note; WCAG2ICT adds 3 more

Not an issue: When WCAG doesn't have notes, and WCAG adds notes, it's clear enough. Example: [Applying SC 1.3.4 Orientation to Non-Web Documents and Software](https://w3c.github.io/wcag2ict/#applying-sc-1-3-4-orientation-to-non-web-documents-and-software). That said, it might make sense to apply the same improvement uniformly wherever notes have been added.

#### In the glossary

In the glossary, issue (1) is similar to what I described above.

Example (there may be others):

- [keyboard interface](https://w3c.github.io/wcag2ict/#dfn-keyboard-interface) — WCAG has notes 1 and 2; WCAG2ICT has different notes 1 and 2 which do not replace WCAG's notes

Issue (2) is a bit different. (Call this issue 2a.) Currently, added notes appear outside the description list, with different indentation and numbering. It's more clear what's been added, but restarting the numbering without explanation looks like an error.

Examples (there may be others):
- [changes of context](https://w3c.github.io/wcag2ict/#dfn-change-of-context) — WCAG has 1 note and 1 example; WCAG2ICT adds 1 more note
- [relative luminance](https://w3c.github.io/wcag2ict/#dfn-relative-luminance) — WCAG has 6 notes; WCAG2ICT adds 1 more

### Proposed

For issues (1) and (2): Indicate clearly which notes are the added ones. This could be done with text or structure, and optionally enhanced with visual styling.

Initial idea, clumsy but maybe effective: add an `<hr>` between the last WCAG note and the first WCAG2ICT added note. This would only help with issue (2) though.

Better would be a single way of notating the added notes that solves both (1) and (2). This could also address (2a) in the glossary, because it wouldn't be so strange to restart the note numbering if were clearer that it's only for added notes.

**Comment by mitchellevan** [comment]
Screenshot of issue (2), seen in SC [1.4.12 Text Specing](https://w3c.github.io/wcag2ict/#text-spacing)

![WCAG notes 1 and 2 followed by WCAG2ICT notes 1 and 2, which do not replace the WCAG notes](https://github.com/w3c/wcag2ict/assets/3124805/712a2987-6162-45aa-b38e-18cd4782dc2f)

**Comment by mitchellevan** [comment]
Screenshot of issue (2a) in the glossary, seen in [changes of context](https://w3c.github.io/wcag2ict/#dfn-change-of-context)

![Indented "Note" and "Example" followed by outdented "Note 1"](https://github.com/w3c/wcag2ict/assets/3124805/b065d01c-2767-48c9-ad10-1a68f365451f)

**Comment by daniel-montalvo** [comment]
Hi there @mitchellevan 

Thanks much for providing such level of detail for the different issues and examples.

What if we prepend "WCAG2ICT" to all added notes?

We would have:

- "Note #" -- all original WCAG notes.
- "WCAG2ICT Note #" -- explcitly calling out that these are WCAG2ICT specific.

I'd say number restarting wouldn't be much of an issue with this addition.

**Comment by mitchellevan** [comment]
> What if we prepend "WCAG2ICT" to all added notes?

Nice idea @daniel-montalvo, that would solve the issue. 

Could you make this change in a branch and let us see it in a Netlify view? I'm sure it will be noticeable enough, but I could imagine some might find it too prominent and visually distracting. If so, we could fine tune it before committing to main.

> I'd say number restarting wouldn't be much of an issue with this addition.

I agree.

**Comment by maryjom** [comment]
I agree that adding "WCAG2ICT" to our added notes would be an easier and more elegant approach to indicate our notes.  I had hoped in the beginning that we could define a special WCAG2ICT styling for our additions that weren't so visually unappealing (and hard to read) as the dotted underline bold green text. Originally I had used the `<INS>` style on those inserted notes but ultimately removed them because they were so difficult to read.

Some caveats... Since sometimes we repeat WCAG's notes and add our notes, there would have to be a way to specify that this is a WCAG2ICT note in the note class or some other way. In addition, in the few cases where the WCAG2ICT note is a replacement/rewrite of an existing WCAG one, it should likely be done with the text replacement prose to make it clear that the note was replaced.

**Comment by maryjom** [comment]
Daniel has added "WCAG2ICT" to our added notes, but I think this will require some additional verbiage in the "replacement" part to make clear what Notes are actually a replacement for WCAG notes and which are simply additions.

**Comment by daniel-montalvo** [comment]
@maryjom @mitchellevan 

In #457 I took a pass at the sections "Comments by guideline and success criteria" and "Comments by definition".

The PR  builds on three main categories of notes:

- WCAG2ICT -- custom note added in the WCAG2ICT document. Show up as "Note 2 (WCAG2ICT).
- Replacement -- original WCAG note with substitutions according to word substitutions and/or additions agreed for the WCAG2ICT document. Show ups as "Note 1 (Replaced)
- Original -- untouched WCAG note, just copied and pasted from the original WCAG document. Show up as "Note 1 (Original)

I would also appreciate advice in potential style differences for each of these, to make these more clear visually.

Note that numbering does not restart. To me it makes sense, because numbering is done by section. So if we have two WCAG original notes and then one WCAG2ICT added note, WCAG2ICT note would be the third note in the section, and it will have number 3.

Restarting the numbering for each category may be confusing. Imagin you have two original notes (1 and 2) andd then 1 replacement (which would be 1) even if it's the third note in the section. I see this as very confusing.

I wonder if we still need note numbering with this further processing. I know that the 2013 WCAG2ICT document had that, but original WCAG notes in the WCAG documents are not numbered.

**Comment by mitchellevan** [comment]
Great progress! Here are my thoughts about the [preview](https://deploy-preview-457--wcag2ict.netlify.app/) of PR #457

#### "(ADDED)"

The "(ADDED)" marker is working well, both in the SCs and in the glossary.

#### "(REPLACED)"

The "(REPLACED)" marker is problematic.

Example: In 1.4.2 Audio Control, "NOTE 1 (REPLACED)" is a bit awkward. It makes it sound like the whole note was replaced, when actually only some words were replaced.

My recommendation: Omit "(REPLACED)". There are now plenty of instances of "(ADDED)", so by contrast it's easy enough to tell which ones were not added.

Omitting "(REPLACED)" is fine even in an SC with no additions, such as SC 1.4.13 Content on Hover or Focus.

Omitting "(REPLACED)" is fine even in an SC where the single WCAG "NOTE" changes to "NOTE 1", such as SC 2.1.2 No Keyboard Trap.

In 1.4.10 Reflow, "NOTE 2 (REPLACED)" is inaccurate because Note 2 contains no replacements. This will become a non-issue by omitting "(REPLACED)".

#### "(ORIGINAL)"

The "(ORIGINAL)" marker first appears in SC 2.2.1 Timing Adjustable. It is not necessary for clarity, and only adds verbosity.

My recommendation: Omit "(ORIGINAL)".

Omitting "(ORIGINAL)" is fine even in an SC with a mixture of three kinds of notes (original, with replacement, and added). The notes with replacement easily stand out from the notes without replacement because of the bold text in square brackets.

#### Special case: WCAG notes copied twice

This involves SC 2.5.1, 2.5.2, and 2.5.7.

In these SCs, the WCAG notes are copied twice into the "for non-web documents" and "for non-web software" groupings, with different substitutions and sometimes with additions. Among these three SCs it's not consistent which instances of the copied notes are marked as "ADDED".

Recommendation: Treat the first instance of a copied note as not added. Treat subsequent instances of the same copied note as added. In other words:

- 2.5.1: do not mark Note 1 as "ADDED"
- 2.5.2: mark Note 4 as "ADDED"
- 2.5.7: no change needed

**Comment by mitchellevan** [comment]
Regarding your further questions @daniel-montalvo 

> I would also appreciate advice in potential style differences for each of these, to make these more clear visually.

I'm not a visual designer, but the styling seems okay to me, noticeable enough without being distracting.

> Note that numbering does not restart. To me it makes sense, because numbering is done by section. So if we have two WCAG original notes and then one WCAG2ICT added note, WCAG2ICT note would be the third note in the section, and it will have number 3.
> 
> Restarting the numbering for each category may be confusing. Imagin you have two original notes (1 and 2) andd then 1 replacement (which would be 1) even if it's the third note in the section. I see this as very confusing.

I agree that added notes should not start with 1.

> I wonder if we still need note numbering with this further processing. I know that the 2013 WCAG2ICT document had that, but original WCAG notes in the WCAG documents are not numbered.

After I reported #462 today about note numbering, now I see your idea of omitting the numbers. I think I like it. Let's continue this part of the discussion in #462.

**Comment by daniel-montalvo** [comment]
Thanks @mitchellevan 

Seeing no other comments, I've updated this as per your suggestions. See preview at https://deploy-preview-457--wcag2ict.netlify.app/
